app/src/main/java/com/hedvig/app/feature/embark/passages/externalinsurer/retrieveprice/RetrievePriceActivity.kt

```diff
@@ -77,6 +79,7 @@ class RetrievePriceInfoActivity : BaseActivity() {
                     }
                 ) {
                     RetrievePriceScreen(
+                        modifier = Modifier.padding(it),
```

Comment:
nit: can we name `it` `paddingValues`? Quite unclear what `it` is here otherwise unless you see that there is a `Scaffold` above and remember that it provides paddingValues as a parameter to this lambda.

Reply:
Yes